src/main/java/platform/codingnomads/co/springdata/example/mybatis/resultsandresult/Movie.java

```diff
@@ -0,0 +1,18 @@
+package platform.codingnomads.co.springdata.example.mybatis.resultsandresult;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class Movie {
+private Long id;
+private String title;
+private String director;
+private String rating;
+private int lengthMinutes;
```

Comment:
Try to keep you code formatted. It significantly improves readability. 

To make it automated, in [Intellij](https://www.jetbrains.com/idea/guide/tutorials/reformatting-code/reformatting-before-commit/) you can apply formatting and import optimization before commit.